src/createSliderWithTooltip.jsx

```diff
@@ -26,13 +28,25 @@ export default function createSliderWithTooltip(Component) {
       });
     }
     handleWithTooltip = ({ value, dragging, index, disabled, ...restProps }) => {
-      const { tipFormatter } = this.props;
+      const {
+        tipFormatter,
+        tipProps,
+      } = this.props;
+
+      const {
+        prefixCls = 'rc-slider-tooltip',
+        overlay = tipFormatter(value),
+        placement = 'top',
```

Comment:
I think `placement` should be one of restTooltipProps, as for the defaultValue, just use:

```tsx
placement="top"
{...restTooltipProps}
overlay={tipFormatter(value)}	
prefixCls={prefixCls}
overlay={overlay}
```